libcentrifugo/metrics.go

```diff
@@ -19,48 +18,66 @@ import (
 type Metrics struct {
 	// NumMsgPublished is how many messages were published into channels.
-	NumMsgPublished metricCounter `json:"num_msg_published"`
+	NumMsgPublished int64 `json:"num_msg_published"`
 
 	// NumMsgQueued is how many messages were put into client queues.
-	NumMsgQueued metricCounter `json:"num_msg_queued"`
+	NumMsgQueued int64 `json:"num_msg_queued"`
 
 	// NumMsgSent is how many messages were actually sent into client connections.
-	NumMsgSent metricCounter `json:"num_msg_sent"`
+	NumMsgSent int64 `json:"num_msg_sent"`
 
 	// NumAPIRequests shows amount of requests to server API.
-	NumAPIRequests metricCounter `json:"num_api_requests"`
+	NumAPIRequests int64 `json:"num_api_requests"`
 
 	// NumClientRequests shows amount of requests to client API.
-	NumClientRequests metricCounter `json:"num_client_requests"`
+	NumClientRequests int64 `json:"num_client_requests"`
 
 	// BytesClientIn shows amount of data in bytes coming into client API.
-	BytesClientIn metricCounter `json:"bytes_client_in"`
+	BytesClientIn int64 `json:"bytes_client_in"`
 
 	// BytesClientOut shows amount of data in bytes coming out if client API.
-	BytesClientOut metricCounter `json:"bytes_client_out"`
+	BytesClientOut int64 `json:"bytes_client_out"`
 
-	// TimeAPIMean shows mean response time in nanoseconds to API requests. DEPRECATED will return 0
-	TimeAPIMean metricGauge `json:"time_api_mean"`
+	// TimeAPIMean shows mean response time in nanoseconds to API requests. DEPRECATED!
+	TimeAPIMean int64 `json:"time_api_mean"`
 
-	// TimeClientMean shows mean response time in nanoseconds to client requests. DEPRECATED will return 0
-	TimeClientMean metricGauge `json:"time_client_mean"`
+	// TimeClientMean shows mean response time in nanoseconds to client requests. DEPRECATED!
+	TimeClientMean int64 `json:"time_client_mean"`
 
-	// TimeAPIMax shows maximum response time to API request. DEPRECATED will return 0
-	TimeAPIMax metricGauge `json:"time_api_max"`
+	// TimeAPIMax shows maximum response time to API request. DEPRECATED!
+	TimeAPIMax int64 `json:"time_api_max"`
 
-	// TimeClientMax shows maximum response time to client request. DEPRECATED will return 0
-	TimeClientMax metricGauge `json:"time_client_max"`
+	// TimeClientMax shows maximum response time to client request. DEPRECATED!
+	TimeClientMax int64 `json:"time_client_max"`
 
 	// MemSys shows system memory usage in bytes.
-	MemSys metricGauge `json:"memory_sys"`
+	MemSys int64 `json:"memory_sys"`
 
 	// CPU shows cpu usage in percents.
-	CPU metricGauge `json:"cpu_usage"`
+	CPU int64 `json:"cpu_usage"`
+}
 
+// metricsRegistry contains various Centrifugo statistic and metric information aggregated
+// once in a configurable interval.
+type metricsRegistry struct {
 	// mu protects from multiple processes updating snapshot values at once
 	// but raw counters may still increment atomically while held so it's not a strict
 	// point-in-time snapshot of all values.
 	mu sync.Mutex
```

Comment:
This unaligns all the counters below. You said "no need to align" but I don't see why not - as long as you are still dong atomic ops on the ints in the struct a below they will still need to be aligned. 

Perhaps a murex happens to be 8 bytes? Even if so (and my gut feeling is it isn't) that is a platform/implementation detail I don't think is guaranteed so we shouldn't rely on it. 

I think since the bool is gone we can just move this to the end. 

We probably need to include in comments on both these structs reminders not to unaligns them again.